ComponentKit/LayoutComponents/CKStackUnpositionedLayout.mm

```diff
@@ -35,7 +35,7 @@ static CKComponentLayout crossChildLayout(const CKStackLayoutComponentChild &chi
   // stretched children will have a cross dimension of at least crossMin
   const CGFloat childCrossMin = alignItems == CKStackLayoutAlignItemsStretch ? crossMin : 0;
   const CKSizeRange childSizeRange = directionSizeRange(style.direction, stackMin, stackMax, childCrossMin, crossMax);
-  return [child.component layoutThatFits:childSizeRange parentSize:size];
+  return CKComponentComputeLayout(child.component, childSizeRange, size);
```

Comment:
Missed these before I merged. Will make a follow up pull request. 

Sent from my iPhone

> On Mar 25, 2016, at 2:28 PM, Adam Ernst <notifications@github.com> wrote:
> 
> In ComponentKit/LayoutComponents/CKStackUnpositionedLayout.mm:
> 
> > @@ -35,7 +35,7 @@ static CKComponentLayout crossChildLayout(const CKStackLayoutComponentChild &chi
> >    // stretched children will have a cross dimension of at least crossMin
> >    const CGFloat childCrossMin = alignItems == CKStackLayoutAlignItemsStretch ? crossMin : 0;
> >    const CKSizeRange childSizeRange = directionSizeRange(style.direction, stackMin, stackMax, childCrossMin, crossMax);
> > -  return [child.component layoutThatFits:childSizeRange parentSize:size];
> > +  return CKComponentComputeLayout(child.component, childSizeRange, size);
> not necessary, we strip nil components
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>